tools/testing-server/constants.js

```diff
@@ -35,7 +35,5 @@ module.exports.rumFlags = {
   cap: 1,
   spa: 1,
   loaded: 1,
-  sr: 0, // this should be off, for now, if privacy.cookie_enabled is on (default) or Traces tests will fail
-  srf: 1,
-  sre: 1
+  sr: 1 // this should be off, for now, if privacy.cookie_enabled is on (default) or Traces tests will fail
```

Comment:
Does this need to be changed back to `0`?

Reply:
Yes thanks for catching that